src/sentry/lang/java/plugin.py

```diff
@@ -80,6 +80,24 @@ def preprocess_step(self, processing_task):
 
         return True
 
+    def process_exception(self, exception):
+        ty = exception.get("type")
+        mod = exception.get("module")
+        if not ty or not mod:
+            return False
+
+        key = "%s.%s" % (mod, ty)
+
+        for view in self.mapping_views:
+            original = view.lookup(key)
+            if original != key:
+                new_module, new_cls = original.rsplit(".", 1)
```

Comment:
Will this work with generics that contain class paths?

Reply:
No idea. At least not different to how we already do with methods